src/main/java/seedu/address/logic/LoginManager.java

```diff
@@ -5,7 +5,7 @@
  */
 public class LoginManager {
 
-    private static boolean isLoginSuccessful = false;
```

Comment:
Remember to change isLoginSuccessful back to false, if not the login functionality will break :)